medicines/doc-index-updater/tests/delete_manager.rs

```diff
@@ -1,14 +1,14 @@
 extern crate doc_index_updater;
 
-use core::{fmt::Debug, future::Future};
+mod support;
 use doc_index_updater::{delete_manager, service_bus_client::delete_factory};
-use tokio_test::block_on;
+use support::get_ok;
 
 #[test]
 #[ignore]
 fn delete_manager_works() {
-    let sent_message = "This is the test message";
-    let mut delete_client = block_on(delete_factory()).unwrap();
+    let sent_message = "This is the delete test message";
```

Comment:
This makes me think we should enforce the shape of the message somehow instead of using the `Client` directly.

Reply:
Agree there's room for improvement here. Right now I think they're adding value. We might always want tests which only test the `client` calls, in which case we probably could use a `get_test_delete_message`. When we pick up tickets into Discovery, we should add the expected integration test changes into the ACs, in which case the decision to drop less relevant tests can be made when those ACs are met.